scripts/extensions/pool_update.precheck

```diff
@@ -18,12 +19,27 @@ import xcp.logger
 
 
 TMP_DIR = '/tmp/'
-PATCH_PRECHECK_FAILED_UNKNOWN_ERROR = 'PATCH_PRECHECK_FAILED_UNKNOWN_ERROR'
+UPDATE_PRECHECK_FAILED_UNKNOWN_ERROR = 'UPDATE_PRECHECK_FAILED_UNKNOWN_ERROR'
+UPDATE_PRECHECK_FAILED_PREREQUISITE_MISSING = 'UPDATE_PRECHECK_FAILED_PREREQUISITE_MISSING'
+UPDATE_PRECHECK_FAILED_CONFLICT_PRESENT = 'UPDATE_PRECHECK_FAILED_CONFLICT_PRESENT'
+UPDATE_PRECHECK_FAILED_WRONG_SERVER_VERSION = 'UPDATE_PRECHECK_FAILED_WRONG_SERVER_VERSION'
 CANNOT_FIND_UPDATE = 'CANNOT_FIND_UPDATE'
+ERROR_MESSAGE_START = 'Error: '
+ERROR_MESSAGE_END = 'You could try '
+ERROR_MESSAGE_CONFLICT = ' conflicts with '
+ERROR_MESSAGE_PREREQUISITE = 'Requires: '
```

Comment:
The above strings are parsed from `yum install` error messages. Also double confirmed in yum code
https://github.com/rpm-software-management/yum